src/test/compile-fail/rfc-2008-non-exhaustive/enum_missing_wildcard.rs

```diff
@@ -0,0 +1,34 @@
+// Copyright 2012 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+// FIXME issue #44109, PR #45394
+// gate-test-non_exhaustive
+#![feature(non_exhaustive)]
+
+// aux-build:enums.rs
+extern crate enums;
+
+use enums::NormalEnum;
+
+// This errors as we are not handling the wildcard
+// case for enum from external crate marked
+// as non_exhaustive.
+
+fn match_enum(ne: &NormalEnum) -> &str {
+    match *ne {
+        //~^ ERROR `..` required with variant marked as non-exhaustive
```

Comment:
The error message is wrong, the `enum` is marked as non-exhaustive, not the variant

Reply:
Fixed this part of the test.